test/test_runner.go

```diff
@@ -145,6 +145,10 @@ type TestRunner struct {
 	logCollection *LogCollectionHook
 
 	backend *EmulatorBackend
+
```

Comment:
To get these, we will leverage the `flow.json` config file, which contains a relevant mapping, e.g. :
```json
"contracts": {
    "FooContract": "./contracts/FooContract.cdc",
    "ArrayUtils": "./contracts/ArrayUtils.cdc",
    "StringUtils": "./contracts/StringUtils.cdc",
    "ApprovalVoting": "./contracts/ApprovalVoting.cdc"
  },
```